language/de/install.php

```diff
@@ -632,3 +259,155 @@
 
 	'TOPICS_TOPIC_TITLE'			=> 'Willkommen bei phpBB3!',
 ));
+
+// Common navigation items' translation
+$lang = array_merge($lang, array(
+	'MENU_OVERVIEW'		=> 'Übersicht',
+	'MENU_INTRO'		=> 'Einführung',
+	'MENU_LICENSE'		=> 'Lizenz',
+	'MENU_SUPPORT'		=> 'Support',
+));
+
+// Task names
+$lang = array_merge($lang, array(
+	// Install filesystem
+	'TASK_CREATE_CONFIG_FILE'	=> 'Erstelle Konfigurationsdatei',
+
+	// Install database
+	'TASK_ADD_CONFIG_SETTINGS'		=> 'Füge Konfigurationsteinstellungen hinzu',
+	'TASK_ADD_DEFAULT_DATA'			=> 'Lege Standard-Einstellungen in der Datenbank fest',
+	'TASK_CREATE_DATABASE_SCHEMA'	=> 'Erstelle Datenbankstruktur',
+
+	// Install data
+	'TASK_ADD_BOTS'			=> 'Registriere Bots',
+	'TASK_ADD_LANGUAGES'	=> 'Installiere verfügbare Sprachen',
+	'TASK_ADD_MODULES'		=> 'Installiere Module',
+
+	// Install finish tasks
+	'TASK_NOTIFY_USER'			=> 'Sende Benachrichtigungs-Mail',
+	'TASK_POPULATE_MIGRATIONS'	=> 'Rolle Migrationen aus', // TODO: passende Übersetzung
+
+	// Installer general progress messages
+	'INSTALLER_FINISHED'	=> 'Die Installations-Routine wurde erfolgreich beendet',
+));
+
+// Installer's general messages
+$lang = array_merge($lang, array(
+	'MODULE_NOT_FOUND'				=> 'Modul nicht gefunden',
+	'MODULE_NOT_FOUND_DESCRIPTION'	=> 'Ein Modul konnte nicht gefunden werden, da der Service „%s“ nicht definiert ist.',
```

Comment:
Abhängig davon ob wir in dem anderen PR jetzt "Service" oder "Dienst" nehmen sollten wir hier das gleiche nutzen.